spec/models/user_spec.rb

```diff
@@ -2,21 +2,40 @@
 
 RSpec.describe User, type: :model do
   it "is valid with username and password" do
-    user_one = User.create(username: 'rossedfort', password: 'password')
+    user_one = User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024')
     expect(user_one).to be_valid
   end
   it "is invalid without username" do
-    user = User.new(password: 'password')
+    user = User.new(password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024')
     expect(user).to be_invalid
   end
   it "is invalid without password" do
-    user = User.new(username: 'rossedfort')
+    user = User.new(username: 'rossedfort', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024')
```

Comment:
Line is too long. [153/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.